### PR TITLE
BZR-48-Implement-user-data-storage

### PR DESCRIPTION
**Pull request by Oshernee**
Implemented local storage component to store user data and if user is in group.

### CODE REVIEW

File: calendar-app/Components/Footer.js [hunk 1]
```diff
@@ -7,8 +7,6 @@ import GroupScreen from './GroupScreen';
 import HomeScreen from './HomeScreen';
 import CreateEventScreen from './CreateEventScreen';
 
-
-
```
Comment (Lukroza) [inline calendar-app/Components/Footer.js:11]: You should remove these spaces

Comment (Oshernee) [reply calendar-app/Components/Footer.js:11]: There were 2 extra spaces that I removed, you might have understood it incorrectly.

File: calendar-app/Components/Storage/userDataStorage.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    fetchData();
+  }, [userIDKey, inGroupKey]);
+
+  const logError = (action, error) => {
+    console.error(`Error ${action}:`, error);
+  };
+
+  const saveSetting = async (key, value, setter) => {
+    try {
+      await store.save(key, typeof value === 'number' ? value.toString() : value);
```
Comment (Lukroza) [inline calendar-app/Components/Storage/userDataStorage.js:35]: We probably will be storing a UUID Data type, so will this be able to store something like : cba1dd22-5815-4344-b202-914cdc8da27f or "cba1dd22-5815-4344-b202-914cdc8da27f"

Comment (Oshernee) [reply calendar-app/Components/Storage/userDataStorage.js:35]: Yes it should be possible to save the UUID data type as the await store.save(key, typeof value === 'number' ? value.toString() : value); has an option to save string type information which can save the UUID information.

**Review by Lukroza — APPROVED** [review]

**Pull request merged**